recipes-bsp/u-boot/files/0010-dts-iot2050-Support-new-IOT2050-SM-variant.patch

```diff
@@ -0,0 +1,447 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Wed, 29 Nov 2023 23:35:51 +0800
+Subject: [PATCH] dts: iot2050: Support new IOT2050-SM variant
+
+the dts file for IOT2050-SM variant is copied from kernel side without
+any change.
+
+Signed-off-by: Baocheng Su <baocheng.su@siemens.com>
+---
+ arch/arm/dts/Makefile                         |   4 +-
+ arch/arm/dts/k3-am65-iot2050-boot-image.dtsi  |   2 +-
+ .../k3-am6548-iot2050-advanced-sm-u-boot.dtsi |   1 +
+ .../arm/dts/k3-am6548-iot2050-advanced-sm.dts | 346 ++++++++++++++++++
+ board/siemens/iot2050/board.c                 |  15 +-
```

Comment:
You will also have to update `doc/board/siemens/iot2050.rst`, I suspect.

Reply:
Fixed in #509